packages/ui/src/app/pages/Validators/ValidatorList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import { PageLayout } from '@/app/components/PageLayout'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { Statistics } from '@/common/components/statistics'
+import { Era } from '@/validators/components/statistics/Era'
+import { Rewards } from '@/validators/components/statistics/Rewards'
+import { Staking } from '@/validators/components/statistics/Staking'
+import { ValidatorsState } from '@/validators/components/statistics/ValidatorsState'
+
+export const ValidatorList = () => {
+  return (
```

Comment:
Again to reduce the amount of queries to the rpc node, please call `useStakingStatistics` here and pass the results as props to the `ValidatorsState`, `Staking`, `Era`, and `Rewards`.

Reply:
I updated the widgets so now the statistic values are passed as props from parent component.